main.go

```diff
@@ -94,6 +94,10 @@ func main() {
 			setupLog.Error(err, "unable to create webhook", "webhook", "PackageBundle")
 			os.Exit(1)
 		}
+		if err := api.InitActiveBundleValidator(mgr); err != nil {
```

Comment:
Wrap this in the above `if` to disable when running locally as this causes issues due to certificates. Unless you found a workaround?

Reply:
Ah it's the tabs issue.